src/toast/util/toastTypes.ts

```diff
@@ -0,0 +1,25 @@
+export interface ToastData {
+  mode: "danger" | "warning" | "success" | "info";
+  content: React.ReactNode;
```

Comment:
Let's call it a render and it should be a function
```suggestion
  render: () => React.ReactNode;
```